src/App.js

```diff
@@ -8,15 +9,26 @@ class App extends React.Component {
       { title: 'Tab 2', content: 'Some text 2' },
       { title: 'Tab 3', content: 'Some text 3' },
     ],
+
+    activeTab: 'Tab 1',
+  };
+
+  changeActiveTab = (changeTab) => {
```

Comment:
```suggestion
  changeActiveTab = (index) => {
```
если это переменная, которая хранит значение не функцию, тогда название должно отвечать на вопрос: кто, что?

changeTab => изменить вкладку - что сделать?

Reply:
fixed